media/js/common/buckets.js

```diff
@@ -25,7 +25,7 @@ define('buckets', [], function() {
         // Camel-case it.
         property = property[0].toUpperCase() + property.substr(1);
 
-        for (var i = 0, e; e = prefixes[i++];) {
+        for (var i = 0; e; e = prefixes[i++]) {
```

Comment:
@diox is suggesting to just delete this whole file, which goes against the "don't make more than jshint changes", but deleting code works. Got to figure out this file actually is for.